pkg/kademlia/kademlia.go

```diff
@@ -275,16 +277,23 @@ func (k *Kademlia) FetchInfo(ctx context.Context, node pb.Node) (_ *pb.InfoRespo
 	return info, nil
 }
 
-// FindNode looks up the provided NodeID first in the local Node, and if it is not found
-// begins searching the network for the NodeID. Returns and error if node was not found
+// FindNode wraps a call to FindNodeWithVouchers with no vouchers
 func (k *Kademlia) FindNode(ctx context.Context, nodeID storj.NodeID) (_ pb.Node, err error) {
+	defer mon.Task()(&ctx)(&err)
+	return k.FindNodeWithVouchers(ctx, nodeID, nil, nil)
+}
+
+// FindNodeWithVouchers looks up the provided NodeID first in the local Node, and if it is not found
+// begins searching the network for the NodeID. When it contacts nodes, it will send vouchers to them
+// and request to be put into their routing table. Returns and error if node was not found
+func (k *Kademlia) FindNodeWithVouchers(ctx context.Context, nodeID storj.NodeID, vrs []*pb.Voucher, self *pb.Node) (_ pb.Node, err error) {
```

Comment:
do you have to handle any differently, if the node has voucher vs no voucher and trying to find other nodes?

Reply:
Whether it has a voucher or not will determine if the node goes into another node's antechamber. We expect them to have no vouchers if they are brand new nodes